sasmodels/direct_model.py

```diff
@@ -424,6 +426,12 @@ def test_reparameterize():
     derived_Iq = derived_calculator(volume=4*pi/3*Rp*Re**2, eccentricity=Rp/Re)
     assert norm((base_Iq - derived_Iq)/base_Iq) < 1e-13
 
+    # clean up the dll that was created in the cache as part of the test
+    try:
+        os.remove(derived_model.dllpath)
+    except:
```

Comment:
Not required in these cases, but I prefer to avoid trapping SystemExit and KeyboardInterrupt by using `except Exception:` instead of a bare `except:`.